recipe/recipe.yaml

```
@@ -0,0 +1,59 @@
+schema_version: 1
+
+context:
+  name: decaylanguage
+  version: 0.18.6
+  python_min: 3.8
+
+package:
+  name: ${{ name|lower }}
+  version: ${{ version }}
+
+source:
+  url: https://pypi.org/packages/source/${{ name[0] }}/${{ name }}/decaylanguage-${{ version }}.tar.gz
+  sha256: 1bad73238b34ecad658d0184f56ff7f5f71be38b977bded0fee2cd50668a894d
+
+build:
+  number: 1
+  noarch: python
+  script: ${{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python 3.8.*
```

```suggestion
    - python ${{ python_min }}.*
```

Currently there's a bug for all of these that the `python_min` doesn't get properly replaced everywhere, so you need to manually replace the `${{ python_min}}.*` in placed like this.

c.f. https://github.com/conda-incubator/conda-recipe-manager/issues/307